lib/master.js

```diff
@@ -32,7 +33,7 @@ module.exports = function (workers, options) {
 
   function serverStart (callback) {
     serverInstance = serverCreate();
-    serverInstance.listen(PORT, callback);
+    serverInstance.listen(PORT, callback).on('error', errorHandler);
```

Comment:
I wonder if we should be registering the callback handler on the line above, before we call `listen`. Just seems that we could run into a race condition otherwise.

Reply:
Hi @gboysko , good point! I will register the callback before `listen`. Thanks!